sdk/search/azure-search-documents/samples/files/patient_data.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Did you already have samples using medical data? Seems most others have been using the hotels sample. As long as you're consistent, though. If this is the first of new samples, you might consider a consistent model (i.e. hotels) for similarity across languages' samples.

Reply:
Having skimmed the contents as well, I'm a little worried the topic of conversation might be triggering to people who have been impacted by this tragedy. Hotels seem pretty safe.

Reply:
changed it to use hotel data - agree with patient data being sensitive - yes i already had the data for a different project i've been working on